public/js/redux/chartsRedux.js

```diff
@@ -1,28 +1,55 @@
 import { State, Actions, Effect } from 'jumpstate';
-import getChartData from 'services/getChartData';
+import api from 'services/Api';
 import { createYTotalsList, createPartialsList, formattedSeries } from 'helpers/chartsHelpers';
 
 /* ------------- State Management ------------- */
 
 Effect('filterDesk', (filterObj) => {
+    Actions.resetErrors();
     Actions.updateFilter(filterObj);
     Actions.toggleIsUpdatingCharts(true);
-    getChartData(filterObj).then(chartsData => {
-        Actions.toggleIsUpdatingCharts(false);
-        Actions.updateComposerVsIncopy(chartsData);
-    });
+    api.getComposerVsIncopy(filterObj.startDate, filterObj.endDate, filterObj.desk)
```

Comment:
I was thinking about how we could make sure that it was easy to add charts in the future, obviously we only have one for now, but something like this might make sense to increase the ease of adding new charts. Something to think about.

```javascript
// chartList.js
export default chartList = ['composerVsIncopy']

// chartsRedux
import chartList;

chartList.map(chart => {
     api.[`get${chart}`](filterObj.startDate, filterObj.endDate, filterObj.desk)
        .then(chartData => {
            Actions.toggleIsUpdatingCharts(false);
            Actions[`update${chart}`](chartData);
        })
        .catch(error => {
            Actions.toggleIsUpdatingCharts(false);
            Actions.[`get${chart}Failed`](error);
        });
})
```